docsk8s/troubleshooting/ls-k8s-troubleshooting-methods.asciidoc

```diff
@@ -1,35 +1,95 @@
 [[ls-k8s-troubleshooting-methods]]
 === Troubleshooting methods
```

Comment:
```suggestion
=== Troubleshooting tips and suggestions
```

Reply:
"Method" has a specific meaning in computing, so we should use different terminology when possible.